Proposals/xGov-8.md

```diff
@@ -0,0 +1,69 @@
+---
+period: 8
+title: Evil Tools
+author: Joseph Glenn (@LoafPickleWW)
+company_name: Thurstober Digital Studios LLC
+category: Tools
+focus_area: User Onboarding
+open_source: No
+amount_requested: 10000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.

Reply:
> Remember to change this to "Final" if you want your proposal to go through the next voting session.

Done.